docs/en/Running-Hadoop-MapReduce-on-Alluxio.md

```diff
@@ -54,17 +54,6 @@ installation:
 This will allow your MapReduce jobs to recognize URIs with Alluxio scheme (i.e., `alluxio://`) in
 their input and output files.
 
-Optionally, modify `$HADOOP_CLASSPATH` by changing `hadoop-env.sh` also in `conf` directory to
```

Comment:
why remove this part?

Reply:
I tried doing this with hadoop 2.7.2 but the classpath change wasn't getting picked up by the the executors. The approaches described in the next section (-libjars and and distributing the client jar) work better.

Reply:
ack